sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.
+
+For example, this includes:
```

Comment:
Sorry for nitpicking, but I actually want this to be as unambiguous of a charter as possible, to clearly demarcate responsibility within the community.

The charter, as an "official" governing document, should clearly lay out what the scope and terms of the charter are.

My suggestion is to remove "For example", since this is logically equivalent to being an incomprehensive list.

Reply:
I think intent is important here. The intent here is not to be a comprehensive list because:

* The apps space is evolving and changing. As that happens and different areas need a different focus SIG Apps needs to be able to shift or shine a light on something. Other SIGs, suck as docs and auth, need to flex a little as well.
* Members of the steering committee, and even beyond that, have expressed a desire to stop the growth in number of SIGs unless there is a good reason. Some have expressed a desire to shrink the number of SIGs. Rigid boundaries provide an easy way to justify more SIGs rather than do work in an existing SIG. This could easily lead to organizational growth.

We can either leave the examples up or remove the section all together. If the examples are removed from the charter we should put them on the Readme to provide some good communication to people.